src/view/com/util/post-embeds/VideoEmbedInner/VideoWebControls.tsx

```diff
@@ -341,8 +341,8 @@ export function Controls({
           )}
           <ControlButton
             active={muted}
-            activeLabel={_(msg`Unmute`)}
-            inactiveLabel={_(msg`Mute`)}
+            activeLabel={_(msg`Unmute video`)}
+            inactiveLabel={_(msg`Mute video`)}
```

Comment:
The `msg()` function allows an optional `context`, that way translators can know where it's being used and be more specific, without having to change the string displayed to the user

```suggestion
            activeLabel={_(msg({message: `Unmute`, context: 'video'}))}
            inactiveLabel={_(msg({message: `Mute`, context: 'video'}))}
```

Reply:
I had forgotten that was possible with `msg()`, thanks @coreh that's a very helpful suggestion and a better way of doing it! :)